crates/sui-graphql-rpc/src/client/simple_client.rs

```diff
@@ -18,6 +21,50 @@ use crate::{
 };
 
 use super::response::GraphqlResponse;
+
+pub struct GraphqlVariable {
+    pub name: String,
+    pub ty: String,
+    pub value: Value,
+}
+
+pub struct GraphqlVariables {
+    // A list of the variable name to its type
+    pub type_defintions: Vec<(String, String)>,
+    // Maps the variable names to the values
+    pub value_mappings: BTreeMap<String, Value>,
+}
+
+// impl GraphqlVariables {
```

Comment:
Cleanup

Reply:
Good callout, not sure how I missed that